q1.sh

```diff
@@ -1 +1,30 @@
-This is q1 answer
+#a
+awk 'END {print NR}' aliceinwonderland.txt 
+
+#b
+awk '/Alice/ {count++} END {print count}' aliceinwonderland.txt
+
+#c
+awk '/Alice/ {count++} END {print count}' aliceinwonderland.txt
```

Comment:
The answer for #c is: 
awk '/ Alice / {count++} END {print count}' aliceinwonderland.txt
I accidently deleted the spaces and as a result it is the same as #b...